chaos-framework/pom.xml

```diff
@@ -95,6 +95,11 @@
             <groupId>com.aliyun.oss</groupId>
             <artifactId>aliyun-sdk-oss</artifactId>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.2</version>
```

Comment:
No need to specific version here, as you specific junit version in parent pom

Reply:
ok，I‘ll try to write it int parent pom.